### ISSUE TITLE
Graph view is not working in 2.2.x update

### ISSUE THREAD

**Issue by DX5536** [YarnSpinnerTool/VSCodeExtension#8]
I updated the Yarn extension from 2.0.0 to 2.2.x kinda breaks VS Code (especially the graph doesn't show up anymore). I recorded a video to showcase the issue here:

https://user-images.githubusercontent.com/49788345/173207240-f07e8bc3-e592-4097-b0a7-b7c8e2c35350.mp4

**Comment by Kaosumaru** [comment]
Also encountered this issue. I think that problem is with
```ts
        this.executeCommand<NodeInfo[]>(
            Commands.ListNodes,
            document.uri.toString())
            .then(result => {
                updateWebView(result);
            })
```

`document.uri.toString()` returns urlencoded string like `file:///v%3A/unity/...`, which is misinterpreted by language server.
Probably extension development is on mac/linux, and you don't have these pesky : in paths? :)

**Comment by Kaosumaru** [comment]
Replacing `document.uri.toString()` with `document.uri.fsPath` fixes the issue :)
Will try to create PR later on...

**Comment by Kaosumaru** [comment]
Not sure, if it fixed all the issues, though - after adding a new node, I don't see it added immediately to the graph view. 
(and I suppose that it should work this way?)

**Comment by Kaosumaru** [comment]
Same with deletion - text document is being modified, but it seems that graph view is not updated.

**Comment by Kaosumaru** [comment]
OK, second issue is in
```ts
        const onNodesChangedSubscription = this.onDidChangeNodes((params) => {
            if (params.uri == document.uri) {
                updateWebView(params.nodes);
            }
        })
```
and is more fun one.

AFAIK `params.uri` is typed as `Uri`, but it's actually a `string` (since it's probably a result of a json deserialization, or things like that)

**Comment by Kaosumaru** [comment]
Moreover, that string is not a valid Uri on windows and that comparison is also broken, as `document.uri` is an object.
Without changing language server, best fix that I can think of rn is
```ts
        const onNodesChangedSubscription = this.onDidChangeNodes((params) => {
            let paramsPath = params.uri.replace("file://", "")
            let paramsUri = vscode.Uri.file(paramsPath);
            if (paramsUri.fsPath == document.uri.fsPath) {
                updateWebView(params.nodes);
            }
        })
 ```

**Comment by desplesda** [comment]
We’ve fixed this issue in v2.1.12, which is now available as a Pre-Release version of the extension. Big thanks to @Kaosumaru for identifying the fix and submitting the PR for it!
